src/CacheKernel.php

```diff
@@ -1,11 +1,24 @@
 <?php
+
 declare(strict_types=1);
 
 /*
  * This file is part of the package t3o/gettypo3org.
  *
+ * This program is free software; you can redistribute it and/or modify
```

Comment:
I would personally prefer referencing to the LISENCE.md as all files are under the same license.

Reply:
Thanks for your input. That's the minimal header suggested by the GPL see appendix. The license file is mentioned on the before last line. I just did it almost the same way we also do in the core.